### ISSUE TITLE
when auth providers groups are added in the cluster RBAC, the page goes blank

### ISSUE THREAD

**Issue by anupama2501** [rancher/dashboard#4271]
Setup

Rancher version: 2.6.1-rc2
Browser type & version: Chrome

**Describe the bug**
If we add auth providers groups as cluster members/owners/ custom permissions, the RBAC page goes blank with the console error `Cannot read properties of undefined (reading 'toLowerCase')`

**To Reproduce**
1. Create an rke1 or rke2 [cluster type doesn't matter, local cluster can be used as well] 
2. Enabled openLDAP auth with the option `Search direct and nested groups` radio button in the auth page while enabling
3. From the explore cluster --> select the cluster --> RBAC --> add a few users 
4. Add a group as a cluster owner/member/custom membership and click on save

**Result**
Page goes blank and no previously added users seen in the RBAC page
Post request to the API server is successful.
Errors seen in console logs:
```
vue.runtime.esm.js:1888 TypeError: Cannot read properties of undefined (reading 'toLowerCase')
    at Proxy.nameSort (resource-instance.js:342)
    at Object.get (resource-proxy.js:93)
    at w (object.js:62)
    at sort.js:190
    at Array.sort (<anonymous>)
    at n.sortBy (sort.js:187)
    at f.arrangedRows (sorting.js:44)
    at _n.get (vue.runtime.esm.js:4479)
    at _n.evaluate (vue.runtime.esm.js:4584)
    at f.arrangedRows (vue.runtime.esm.js:4836)
```

**Expected Result**
The page shouldn't go blank and no errors seen in the console logs

**Screenshots**
<img width="500" alt="Screen Shot 2021-09-23 at 11 48 31 AM" src="https://user-images.githubusercontent.com/38144301/134574061-5521a367-8b53-4f11-9e42-7ef87e255b45.png">

**Comment by ronhorton** [comment]
![Screen Shot 2021-09-27 at 12.47.03 PM.png](https://zube.io/files/rancher/9a196326d6c109590a278810721a6332-screen-shot-2021-09-27-at-12-47-03-pm.png)

**Comment by ronhorton** [comment]
pass - verified fixed in v2.6-fe57abd77598577487eb5722aba7bc1eebb50d70-head
The screenshot I attached is from the test, showing that the page is not blank.
